arch/risc-v/src/mpfs/mpfs_ihc.c

```diff
@@ -177,8 +177,10 @@ static struct mpfs_rptun_shmem_s    g_shmem;
 static struct rpmsg_device         *g_mpfs_rpmsg_device;
 static struct rpmsg_virtio_device  *g_mpfs_virtio_device;
 
+#ifndef CONFIG_MPFS_OPENSBI
 static sem_t  g_mpfs_ack_sig       = SEM_INITIALIZER(0);
 static sem_t  g_mpfs_rx_sig        = SEM_INITIALIZER(0);
```

Comment:
If those are signalling semaphores, then maybe it is better to switch to `NXSEM_INITIALIZER(0, PRIOINHERIT_FLAGS_DISABLE);`?

Reply:
ok, changed